internal/tool/ssh/deploy.go

```diff
@@ -228,3 +238,22 @@ func getAbsoluteFilePath(file string) (string, error) {
 	}
 	return abs, nil
 }
+
+// getTmpDirs prints the tmp directory in which the weaver binary will
```

Comment:
getTmpDirs returns the path to the tmp directories where the weaver binaries will be stored at each remote location.